radio/src/gui/480x272/lcd.h

```diff
@@ -133,7 +133,7 @@ void lcdDrawNumber(coord_t x, coord_t y, int32_t val, LcdFlags flags=0, uint8_t
 
 void putsModelName(coord_t x, coord_t y, char *name, uint8_t id, LcdFlags att);
 void putsStickName(coord_t x, coord_t y, uint8_t idx, LcdFlags att=0);
-void drawSwitch(coord_t x, coord_t y, swsrc_t swtch, LcdFlags flags=0);
+void drawSwitch(coord_t x, coord_t y, int32_t swtch, LcdFlags flags=0);
```

Comment:
I can't seem to recreate the original warning anymore (and don't remember exactly which combo of options/compilers/targets caused it to begin with).  So if you guys would prefer to go with the original extern declaration, I can just revert this commit.

Should then also maybe change, for consistency:
```diff
--- a/radio/src/gui/480x272/lcd.h  (and in 212x64/lcd.h)
+++ b/radio/src/gui/480x272/lcd.h
@@ -137,1 +137,1 @@
- void drawSource(coord_t x, coord_t y, uint32_t idx, LcdFlags att=0);
+ void drawSource(coord_t x, coord_t y, mixsrc_t idx, LcdFlags att=0);
```